Jenkinsfile

```diff
@@ -54,6 +57,7 @@ pipeline {
                                             "-p",
                                             "TAG=${env.TAG}",
                                             "IMAGESTREAM_TAG=${params.IMAGE_STREAM_LATEST_TAG}",
+                                            "REGISTRY_PROJECT=${params.REGISTRY_PROJECT}",
```

Comment:
Good addition! I was working on something similar but yours looks much better. I'd only suggest documenting the role of `REGISTRY_PROJECT` a bit more